### PR TITLE
Update energy_system_sector_selector.rst

### PR DESCRIPTION
**Pull request by srhbrnds**
**Changes proposed in this pull request**:
- resolved formatting issues
- fixed typos
- provide minor changes to the Requirements of the view-component

### CODE REVIEW

File: docs/view_components/energy_system_sector_selector.rst [hunk 1]
```diff
@@ -10,35 +10,36 @@ Attributes
...
+        * each sector has an associated icon
 
 **list of selected energy sectors**
     Contains the choices selected by the user from the `es_sector_avail_list`
+    
     Properties:
-        - id `es_sector_select_list`
-        - each sector has an associated :ref:`energy-type-def`
+        * id `es_sector_select_list`
+        * each sector has an associated :ref:`energy-type-def`
```
Comment (Bachibouzouk) [inline docs/view_components/energy_system_sector_selector.rst:22]:
```suggestion
        * each sector has an associated :ref:`energy-type-def`
        * each sector has an associated icon
```

File: docs/view_components/energy_system_sector_selector.rst [hunk 2]
```diff
@@ -10,35 +10,36 @@ Attributes
...
-    Indicates whether there is a coupling between two sector
+    Indicates whether there is a coupling between two sectors
 
     Properties:
-        - id `es_sector_coupling_list`
-        - each element is a list of two interconnected sectors
-        - an empty list means that all sectors are independent
+        * id `es_sector_coupling_list`
+        * each element is a list of two interconnected sectors
+        * an empty list means that all sectors are independent
```
Comment (Bachibouzouk) [inline docs/view_components/energy_system_sector_selector.rst]:
```suggestion
        * an empty list means that all sectors are independent from each other
```

**Review by Bachibouzouk — APPROVED** [review]
See if you think my suggestion make sense, then you can merge :)

**Review by mahendrark — APPROVED** [review]

File: docs/view_components/energy_system_sector_selector.rst [hunk 3]
```diff
@@ -49,4 +50,4 @@ Link with other view-components
 Rendering of the view-component
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Each sector in the list is represented by its icon. The energy sectors in the `es_sector_select_list` are rendered in color and the remaining energy sectors (in `es_sector_avail_list` but not in `es_sector_select_list`) are rendered in shades of grey.
-Each sector is visually connected to all the other sectors by a greyed-out line (no active sector coupling), If the line is display in bright color (e.g. after an activating click on it by the user) it means the two sectors are interconnected directly.
\ No newline at end of file
+Each sector is visually connected to all the other sectors by a greyed-out line (no active sector coupling), If the line is display in bright color (e.g. after an activating click on it by the user) it means the two sectors are interconnected directly.
```
Comment (TheOneAndra) [inline docs/view_components/energy_system_sector_selector.rst]:
```suggestion
Each sector is visually connected to all the other sectors by a greyed-out line (no active sector coupling). If the line is display in bright color (e.g. after an activating click on it by the user) it means the two sectors are interconnected directly.
```

**Review by TheOneAndra — APPROVED** [review]

**Comment by Bachibouzouk** [comment]
@srhbrnds - when you merge, please click on "delete branch" button :)

**Pull request merged**